kebechet/cli.py

```diff
@@ -88,7 +90,22 @@ def cli_run_results(origin, service, analysis_id):
 @click.option("-s", "--service", envvar="KEBECHET_SERVICE_NAME")
 def cli_run_url(url, service):
     """Run Kebechet by providing url to a git repository service."""
-    config.run_url(url, service)
+    config.run_url(url, service, None)
+
+
+@cli.command("run-webhook")
+@click.argument("web_payload", nargs=1)
+def cli_run_webhook(web_payload):
+    """Run Kebechet by providing a webhook payload."""
+    payload = None
+    if os.path.isfile(web_payload):
+        with open(web_payload) as f:
+            payload = json.load(f)
+    else:
+        # If the json is passed a string.
+        payload = json.loads(web_payload)
+    if payload:
+        config.run_webhook(payload)
```

Comment:
why only if payload is not null? Should the program end silently with no operation if no payload is present?

Reply:
Should we raise an exception?

Reply:
Yes - if there is no payload, there is nothing to do.

Reply:
Added an exception :)